python/strings_udf/strings_udf/__init__.py

```diff
@@ -87,6 +88,29 @@ def _get_ptx_file():
         return regular_result[1]
 
 
+# Maximum size of a string column is 2gb
+STRINGS_UDF_DEFAULT_HEAP_SIZE = int(2e9)
+STRINGS_UDF_HEAP_SIZE = os.environ.get(
+    "STRINGS_UDF_HEAP_SIZE", STRINGS_UDF_DEFAULT_HEAP_SIZE
+)
+heap_size = 0
+
+
+def set_malloc_heap_size(size=STRINGS_UDF_HEAP_SIZE):
+    """
+    Heap size control for strings_udf, size in bytes.
+    """
+    global heap_size
+    if size == heap_size:
+        return
+    else:
+        (ret,) = cudart.cudaDeviceSetLimit(cudart.cudaLimit(2), size)
```

Comment:
I think this is what you need. Access the enum value by name, rather than explicitly referring to its integer equivalent.
```suggestion
        (ret,) = cudart.cudaDeviceSetLimit(cudart.cudaLimit.cudaLimitMallocHeapSize, size)
```